commit 4184b4b09f4ff6bd630657dc51bb25c899b9df9a (HEAD -> master, refs/changes/40/1940/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-05-11T15:35:34-07:00 (2 years, 5 months ago)
    
    RELENG-760 some repos are obsolete, mark them as such and drop them from devguide

diff --git a/README.md b/README.md
index c3b552f..5197883 100644
--- a/README.md
+++ b/README.md
@@ -5,9 +5,11 @@
 -->
 
 <!--
-    Copyright (c) 2014, Joyent, Inc.
+    Copyright (c) 2017, Joyent, Inc.
 -->
 
+**Note: This repository is obsolete.**
+
 # SDC-UFDS-Replicator
 
 This repository is part of the Joyent SmartDataCenter project (SDC).  For
